course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+# EEB430H1 Theoretical Ecology [24L, 12P]
+Theoretical ecology uses models to explain biological phenomena such as the maintenance of biodiversity, population growth, competition, eco-evolutionary dynamics, epidemiology, spatial ecology, and species extinction. This course is designed to meet a growing demand for reproducible, openly accessible, analytically thorough, and well documented science. Students will learn to develop ecological population models, analyze data and document their research using the R programming language. No prerequisite programming experience is required.
+
+Prerequisites: MAT 136, BIO220H1, EEB225H1, and two of EEB319H1, EEB321H1,
```

Comment:
Was the final word that we were just supposed to keep whatever prereqs they already listed?